indico/modules/oauth/__init__.py

```diff
@@ -22,3 +23,5 @@ def _extend_admin_menu(sender, **kwargs):
 @signals.menu.items.connect_via('user-profile-sidemenu')
 def _extend_profile_sidemenu(sender, user, **kwargs):
     yield SideMenuItem('applications', _('Applications'), url_for('oauth.user_apps'), 40, disabled=user.is_system)
+    if can_manage_personal_tokens() or user.personal_tokens.filter_by(revoked_dt=None).has_rows():
```

Comment:
What do you think about adding a `active_tokens` or similar column property to `User´? This would avoid repeating the `revoked_dt` filter.

Reply:
yeah, I started thinking the same when writing this code...

Reply:
updated; i added a util instead of a second relationship since overlapping relationships are a bit ugly (and fragile)